source/patterns/@aws-solutions-constructs/aws-cloudfront-mediastore/test/integ.existingContainer.ts

```diff
@@ -15,13 +15,14 @@
 import { App, Stack } from '@aws-cdk/core';
 import * as mediastore from '@aws-cdk/aws-mediastore';
 import { CloudFrontToMediaStore } from '../lib';
+import { generateIntegStackName } from '@aws-solutions-constructs/core';
 
 // Setup
 const app = new App();
-const stack = new Stack(app, 'test-cloudfront-mediastore');
+const stack = new Stack(app, generateIntegStackName(__filename));
 stack.templateOptions.description = 'Integration test for aws-cloudfront-mediastore with existing mediastore container';
 const mediaStoreContainerObject = new mediastore.CfnContainer(stack, 'MyMediaStoreContainer', {
-  containerName: 'MyMediaStoreContainer'
+  containerName: 'MyMediaStoreContainer2'
```

Comment:
Why? To avoid a name collision? If so, let's change one or the other name to clarify the difference. Just adding "2" is seldom the best way forward. (It's right up there with Microsoft adding "Ex" to improved API calls).

Reply:
Yep to avoid the name collision

Reply:
Renamed to `MyExistingMediaStoreContainer`